web_api/__init__.py

```
@@ -0,0 +1,319 @@
+    api = ProfileAPI(
+        profile_store,
+        config_manager,
+        conversation_ledger,
+        chat_sources,
+        status_transition_manager,
+        debounce_manager,
+        last_decisions,
+    )
+
+    routes = [
+        (f"/astrbot_plugin_angel_heart/profiles", api.list_profiles, ["GET"], "模板列表与绑定总览"),
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)